expression/builtin_math.go

```diff
@@ -190,8 +192,85 @@ func builtinRound(args []types.Datum, ctx context.Context) (d types.Datum, err e
 
 // See http://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_conv
 func builtinConv(args []types.Datum, ctx context.Context) (d types.Datum, err error) {
-	//TODO  implement
-	return d, errors.New("Function unimplement")
+	var (
+		n        string
+		toBase   int64
+		fromBase int64
+		signed   bool
+		negative bool
+		touval   bool
+	)
+	for _, arg := range args {
+		if arg.IsNull() {
+			return d, nil
+		}
+	}
+	n, err = args[0].ToString()
```

Comment:
Use `:=` instead of `=`, and remove line 196.
The same as line 213, line 217 and line 233.